00-Raster-Structure.Rmd

```diff
@@ -460,20 +449,20 @@ is likely caused by an error in either data collection or processing.
 
 ### Find Bad Data Values
 Sometimes a raster's metadata will tell us the range of expected values for a
- raster. Values outside of this range are suspect and we need to consider than
- when we analyze the data. Sometimes, we need to use some common sense and
- scientific insight as we examine the data - just as we would for field data t
- identify questionable values. 
+raster. Values outside of this range are suspect and we need to consider than
+when we analyze the data. Sometimes, we need to use some common sense and
+scientific insight as we examine the data - just as we would for field data to
+identify questionable values. 
 
 ## Create A Histogram of Raster Values
 
 We can explore the distribution of values contained within our raster using the 
-`hist` function which produces a histogram. Histograms are often useful in 
+`hist()` function which produces a histogram. Histograms are often useful in 
```

Comment:
@brymz I like this approach of using `hist()` (vs `hist`). @mjones01 and i went back and forth on it as i was using the latter and she was using the former. Do you think we should do this throughout for all functions? If so, i'll post an issue to address in our other lesson repos too!